<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAssetsUI</class>
 <widget class="QWizard" name="FindAssetsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Assets</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Let's find the assets here!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="UnzipPage" name="wizardPage2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>First we could download the version manifest file.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Choose the download directrory, or use the cache directory.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="setBtn">
        <property name="text">
         <string>Set to the cache dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openBtn">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="manifestBtn">
      <property name="text">
       <string>Download version manifest</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>The version manifest is invalid!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Then you can choose the game directory to download the file, Or leave empty to use the cache directory.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="openBtn_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Choose a version.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="versionBtn">
      <property name="text">
       <string>Download version json</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Assets index file:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="indexLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="indexBtn">
      <property name="text">
       <string>Download assets index file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gameBtn">
      <property name="text">
       <string>Download game cilent jar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Now you can choose the assets file to copy from, and extract the game jar to get extra information.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="assetsBtn">
      <property name="text">
       <string>Select Assets File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="unzipBtn">
      <property name="text">
       <string>Unzip Resource Jar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UnzipPage</class>
   <extends>QWizardPage</extends>
   <header>unzippage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
